portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -10,6 +10,7 @@
 
 @WebServlet("/login")
 public class HomeServlet extends HttpServlet {
+  public static String userEmail;
```

Comment:
Why is it static? Is there only 1 email address?

Reply:
It's static because there's only one current email address. I renamed `userEmail` to `currentUserEmail` so hopefully it's more intuitive.

I originally had it there so I could have access to the email in a different java class. I just removed it though because I think there will be less issues if I just retrieve the email again in the different java class.

Done!